src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -686,12 +760,16 @@ def _input_handler():
             run_parameters["nts"] = args.nts
             run_parameters["qts_subdivisions"] = args.qts_subdivisions
             run_parameters["compute_method"] = args.compute_method
-
             waterbody_parameters[
                 "break_network_at_waterbodies"
             ] = args.break_network_at_waterbodies
             output_parameters["csv_output_folder"] = args.csv_output_folder
-
+            data_assimilation_parameters[
+                "data_assimilation_parameters_file"
+            ] = args.data_assimilation_parameters_file
+            data_assimilation_filter[
+                "data_assimilation_filter"
+            ] = args.data_assimilation_filter
             restart_parameters[
```

Comment:
add a space after line 772 to separate `data_assimilation_parameters` from `restart_parameters.`